test/e2e/e2e_suite_test.go

```diff
@@ -143,6 +181,24 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	var err error
 	e2eConfig, err = vsphereframework.LoadE2EConfig(ctx, configPath)
 	Expect(err).NotTo(HaveOccurred())
+	if configOverridesPath != "" {
+		Expect(configOverridesPath).To(BeAnExistingFile(), "Invalid test suite argument. e2e.config-overrides should be an existing file.")
+
+		Byf("Merging with e2e config overrides from %q", configOverridesPath)
+		configData, err := os.ReadFile(configOverridesPath) //nolint:gosec
+		Expect(err).ToNot(HaveOccurred(), "Failed to read e2e config overrides")
+		Expect(configData).ToNot(BeEmpty(), "The e2e config overrides should not be empty")
+
+		byoConfig := &byoConfigOverrides{}
```

Comment:
```suggestion
		configOverrides := &configOverrides{}
```
Maybe there are more occurences of 'byo' elsewhere